include/zephyr/drivers/usb/udc.h

```diff
@@ -18,6 +18,15 @@
 #include <zephyr/sys/atomic.h>
 #include <zephyr/usb/usb_ch9.h>
 
+/**
+ * @brief USB device controller (UDC) API
+ * @defgroup udc_api USB device controller API
+ * @ingroup io_interfaces
+ * @since 3.3
+ * @version 0.2.0
+ * @{
+ */
```

Comment:
I am not yet convinced to remove experimental for this release, but immediately after. But it is more work, there are many places where "experimental" needs to be removed and documentation needs to be updated.

There are few more headers to include API version, see https://github.com/zephyrproject-rtos/zephyr/pull/80385